Fix Logo aspect ratio on Privacy Center `404` page and Consent Page

RobertKeyser opened the pull request:
Closes #2893 

### Code Changes

* Replace 404 logo width/height attributes to maxWidth/maxHeight
* Increase the maxWidth/maxHeight sizes to make the logo appear larger on 404 page
* Remove width constraint from consent page

### Steps to Confirm

* Launch privacy center from main (`npm run dev`)
* Replace the logo in the `config.json` with a few others:
  * https://upload.wikimedia.org/wikipedia/commons/4/46/Wikipedia-logo-v2-en_6m_articles_204_white.png
  * https://upload.wikimedia.org/wikipedia/commons/8/85/Logo-Test.png
* browse to http://localhost:3000/fake-page
* browse to consent page
* Launch privacy center from this branch (`rk-2893-privacy-center-404`)
* Repeat testing steps to confirm logos look correct

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [x] Issue Requirements are Met
* [x] Relevant Follow-Up Issues Created
* [x] Update `CHANGELOG.md`
* [x] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### Description Of Changes

_Write some things here about the changes and any potential caveats_

ThomasLaPiana commented:
@RobertKeyser do you have screenshots of the diff? We usually prefer to include those for UI changes

RobertKeyser commented:
@ThomasLaPiana thanks for the suggestion

Original 404:
![image](https://user-images.githubusercontent.com/39230492/227234095-a1768b26-837b-4a27-9a18-361fa0dd71b6.png)

Fixed 404:
![image](https://user-images.githubusercontent.com/39230492/227235043-0186fcd5-ade7-4354-bcec-caa711693c94.png)

Original Consent:
![image](https://user-images.githubusercontent.com/39230492/227232954-63a576fc-add1-4493-bd93-85155d145c55.png)

Fixed Consent:
![image](https://user-images.githubusercontent.com/39230492/227231257-8a8b994f-afa2-4a49-8fd0-ba345d039796.png)

seanpreston reviewed (approved):
Thanks @RobertKeyser this works a charm:

![Screenshot 2023-03-23 at 12 54 00](https://user-images.githubusercontent.com/1667340/227277597-032a190a-2097-4d27-8f4b-fa17681e2eca.png)

![Screenshot 2023-03-23 at 12 54 13](https://user-images.githubusercontent.com/1667340/227277659-875d41a6-4dc2-4e0d-8a25-7a1fe0cada8e.png)

I do think there's another conversation we can have in the design loop with regards to how much real estate the privacy centre gives the logos, and this is a great first step.
